packages/server-wallet/src/wallet/multi-threaded-wallet/manager.ts

```diff
@@ -6,37 +6,38 @@ import {UpdateChannelParams} from '@statechannels/client-api-schema';
 import {Either} from 'fp-ts/lib/Either';
 import {isLeft} from 'fp-ts/lib/These';
 import _ from 'lodash';
+import {Logger} from 'pino';
 
 import {ServerWalletConfig} from '../../config';
-import {logger as parentLogger} from '../../logger';
+import {createLogger} from '../../logger';
 import {MultipleChannelOutput, SingleChannelOutput} from '../wallet';
 
 import {StateChannelWorkerData} from './worker-data';
 
 const ONE_DAY = 86400000;
 
-const logger = parentLogger.child({module: 'Worker-Manager'});
-
 export class WorkerManager {
   private pool?: Pool<Worker>;
   private threadAmount: number;
+  private logger: Logger;
   constructor(walletConfig: ServerWalletConfig) {
+    this.logger = createLogger(walletConfig).child({module: 'Worker-Manager'});
     this.threadAmount = walletConfig.workerThreadAmount;
     if (this.threadAmount > 0) {
       this.pool = new Pool({
         create: (): Worker => {
-          logger.trace('Starting worker');
+          this.logger.trace('Starting worker');
 
           const worker = new Worker(path.resolve(__dirname, './loader.js'), {
             workerData: walletConfig,
           });
 
-          worker.stdout.on('data', data => logger.info(data.toString()));
-          worker.stderr.on('data', data => logger.error(data.toString()));
+          worker.stdout.on('data', data => this.logger.info(data.toString()));
+          worker.stderr.on('data', data => this.logger.error(data.toString()));
```

Comment:
If `data` is a stringified object e.g., `"{type:'debug',msg:'blah',channelId:'0xabc'}"` would `this.logger.info` interpret it as an object or a string?

Reply:
This explains some noisy and unreadable logs I've seen. What is even the purpose of this?

Reply:
I think the purpose of this is to capture logging output from the `worker` when `LOG_DESTINATION=console`. However it looks like the parent `stdout` gets passed in by default so this probably just messes up the logs.

Reply:
I think the problem would be solved if the worker had the same logging config as the main thread.
- if it's `console`, then there's no need to print the same log line twice, once in an unreadable format
- if the main thread's logger logs to a filename, then so should the worker thread's right?